### ISSUE TITLE
Adding data to the Emojicon Map

### ISSUE THREAD

**Issue by AljoSt** [rockerhieu/emojicon#132]
Hey, I was wondering, if it is possible to add custom "rules" to the Emojicon-Map. Right now, Emojicons like :-) are not converted to proper images. It would be good, if it would be possible to have a method like `addSubstitution(String original, String subsitution)` ->   `addSubstitution(":-)", _correspondingUnicode_)`

**Comment by rockerhieu** [comment]
Yes it is possible but it will impact the performance a lot if the custom map is getting bigger.
